collections/_article/maintenance-release-godot-4-1-2.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Maintenance release: Godot 4.1.2"
+excerpt: "As Godot 4.2 enters feature freeze this week, a new Godot 4.1 maintenance release arrives with a handful of improvements to stability and performance!"
+categories: ["release"]
+author: Yuri Sizov
+image: /storage/blog/covers/maintenance-release-godot-4-1-2.webp
+image_caption_title: Robot Detour
+image_caption_description: A game by Nozomu Games
+date: 2023-10-04 14:00:00
+---
+
+Godot 4.1 is celebrating its [3 month anniversary](/article/godot-4-1-is-here) this week, and that means Godot 4.2 is entering the beta testing phase shortly after. Each development cycle is fruitful with bugfixes and other sorts of improvements, and it's an integral part of our development process to identify such changes. If these contributions prove to be safe and stable, we compile a bunch of them into a patch release for the current stable version of the engine.
```

Comment:
Just a grammar and style suggestion that simplifies things a bit but gets the same point across. 

```suggestion
Godot 4.1 is celebrating its [3 month anniversary](/article/godot-4-1-is-here) this week, and that means Godot 4.2 is entering the beta testing phase shortly after. This development cycle has been packed with bugfixes and other  improvements. As always, we take the safest of these contributions and we compile them into a patch release for the current stable version of the engine.
```

Reply:
Thanks, that's much better!